ddtrace/internal/ci_visibility/constants.py

```diff
@@ -0,0 +1,7 @@
+EVP_PROXY_AGENT_BASE_PATH = "evp_proxy/v2/"
+EVP_PROXY_AGENT_ENDPOINT = "{}api/v2/citestcycle".format(EVP_PROXY_AGENT_BASE_PATH)
```

Comment:
nitpick: this is a requirement for the suggestion below. This way you can "forget" about leading and trailing slashes when checking evp proxy compatibility 

```suggestion
EVP_PROXY_AGENT_BASE_PATH = "evp_proxy/v2"
EVP_PROXY_AGENT_ENDPOINT = "{}/api/v2/citestcycle".format(EVP_PROXY_AGENT_BASE_PATH)
```